support/openharmony/entry/build-profile.json5

```diff
@@ -0,0 +1,37 @@
+{
+  "apiType": "stageMode",
+  "buildOption": {
+    "arkOptions": {
+      // "apPath": "./modules.ap"  /* Profile used for profile-guided optimization (PGO), a compiler optimization technique to improve app runtime performance. */
```

Comment:
I guess this could be removed as well.

Reply:
I guess this is generated by the IDE. I've removed it in [ac570b1](https://github.com/servo/servo/pull/33295/commits/ac570b1037d07be31689e2819284ec4648e039ae).